taskcluster/ci/ui-test-apk/kind.yml

```diff
@@ -103,6 +103,7 @@ tasks:
             shipping-product: focus
         description: 'Focus UI tests with firebase'
         run-on-tasks-for: [github-pull-request, github-push]
+        run-on-git-branches: ["^((?!releases[_/]).+)$"]
```

Comment:
Previously, this was running on `releases` branches (e.g, https://github.com/mozilla-mobile/firefox-android/runs/12386003579). To match `Fenix`, we want this to run when Focus changes happen elsewhere.